torch_np/_detail/_util.py

```diff
@@ -111,7 +111,7 @@ def expand_shape(arr_shape, axis):
 def apply_keepdims(tensor, axis, ndim):
     if axis is None:
         # tensor was a scalar
-        tensor = torch.full((1,) * ndim, fill_value=tensor, dtype=tensor.dtype)
+        tensor = torch.full((1,) * ndim, fill_value=tensor.item(), dtype=tensor.dtype)
```

Comment:
So is it  `tensor.expand(shape).clone()` ?

Reply:
`tensor.expand(shape).contiguous()`. Note that if it already has the correct shape, this will not copy.
If you *always* want a fresh copy, do `tensor.expand(shape).clone(memory_format=torch.contiguous_format)`.

Reply:
OK, thanks!